client/blocks/signup-form/passwordless.jsx

```diff
@@ -25,8 +25,8 @@ class PasswordlessSignupForm extends Component {
 	static propTypes = {
 		locale: PropTypes.string,
 		inputPlaceholder: PropTypes.string,
-		submitButtonLabel: PropTypes.string,
-		submitButtonLoadingLabel: PropTypes.string,
+		submitButtonLabel: PropTypes.oneOfType( [ PropTypes.string, PropTypes.node ] ),
```

Comment:
I wonder why this change is needed? I'm not seeing any consumers passing a node 🤔

Reply:
Because we changed the `submitButtonLoadingLabel`, I think it would be a bit odd/confusing for these two to have different prop types. So, I changed this to ensure consistency. Does that sound good?

Reply:
I see. That makes sense, LGTM!